### PR TITLE
Aiden Live workspace and 0.41.2

### PR DESCRIPTION
**Pull request by sambitcreate**
## Summary
- restore the app/Assistant mark for the one-time Live setup trigger and rename the user-facing feature to Aiden Live (Beta)
- give every Live start a fresh metadata-only backend thread, with no ordinary Assistant chat, transcript, audio, screenshot, or prompt persistence
- keep Computer Use main-authorized and approval-gated, add actionable Live approvals, and expose listening/thinking/speaking/acting/approval/error orb states
- make thread begin/finalization crash-safe and shutdown-aware, then bump the release to 0.41.2

## Validation
- `npm run type-check`
- `npm run build`
- `npm test` (1,603 main tests plus all pretest/native/settings suites)
- `npm run test:gemini-live`
- three independent GPT-5.6 Luna edge-case reviews, with findings remediated

## Operator gates
- Native macOS screen-picker and real-Google acceptance receipts remain explicit operator-owned follow-up gates documented in the plan.

### CODE REVIEW

File: renderer/components/assistant/use-assistant-live.ts [hunk 1]
```diff
@@ -866,11 +816,21 @@ export function useAssistantLiveWithDependencies(
     setError(null);
     setReconnectRequired(false);
     setCaptions([]);
+    setComputerUseActing(false);
     try {
+      const computerUseAuthorization = computerUseEnabled
```
[the remark this answers is not in the document]
Comment (sambitcreate) [reply renderer/components/assistant/use-assistant-live.ts:821]: Fixed in 4a64e9b7: `computerUseEnabled` is now part of the `start` callback dependency set, and the hook test asserts the exact one-use authorization token passed to `start` after readiness revalidation.

File: main/services/gemini-live/service-main.ts [hunk 2]
```diff
@@ -28,32 +76,40 @@ export const geminiLiveService = new GeminiLiveService({
     process.env,
     app.getPath("userData"),
   ),
+  threads: {
+    begin: async (input) => {
+      await ensureAidenLiveThreadRecovery();
```
[the remark this answers is not in the document]
Comment (sambitcreate) [reply main/services/gemini-live/service-main.ts:86]: Fixed in 4a64e9b7: normal `app.whenReady()` startup now awaits Aiden Live crash reconciliation before creating the renderer window. I/O failures stay safely logged and the lazy Live lifecycle retry remains in place; corrupt records continue to be preserved and skipped.

File: renderer/components/assistant/use-assistant-live.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (sambitcreate) [reply renderer/components/assistant/use-assistant-live.ts:821]: Fixed in be582682: the renderer now re-checks mounted state, setup abort, and operation generation immediately after Computer Use authorization and before provider start. The new regression test holds authorization pending, cancels setup, resolves authorization, and proves api.start is never called. Focused Aiden Live tests (140 + 46 Computer Use) and type-check pass.
